src/main/java/net/masterthought/cucumber/util/Status.java

```diff
@@ -13,7 +13,8 @@
     SKIPPED("#88AAFF"),
     PENDING("#FBB907"),
     UNDEFINED("#FBB957"),
-    MISSING("#FBB9A7");
+    MISSING("#FBB9A7"),
+    HIDDEN("");
```

Comment:
I can't think of any other place this might go, since it is being used to
override any status of a step if the attribute "hidden" is to be found true
On May 13, 2015 9:41 PM, "Damian Szczepanik" <notifications@github.com>
wrote:

> In src/main/java/net/masterthought/cucumber/util/Status.java
> <https://github.com/masterthought/cucumber-reporting/pull/101#discussion_r30262161>
> :
>
> > @@ -13,7 +13,8 @@
> >      SKIPPED("#88AAFF"),
> >      PENDING("#FBB907"),
> >      UNDEFINED("#FBB957"),
> > -    MISSING("#FBB9A7");
> > +    MISSING("#FBB9A7"),
> > +    HIDDEN("");
>
> HIDDEN is not valid value of status property so it should not be added here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/masterthought/cucumber-reporting/pull/101/files#r30262161>
> .
>